Cyra/lab2/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.14)
+project(nixintro)
+include_directories(include)
```

Comment:
do not use include_directories, only use target_include_directories and include directories per each target, not globally within a cmakelists